src/assets/index.ts

```diff
@@ -0,0 +1,8 @@
+export const images = {
```

Comment:
Nice! This was a clever way of getting around dynamically using images in the toolbar

Reply:
I remembered you telling me about it before so I tried removing this and it broke. Maybe you can look at this with me later

Reply:
Yeah for sure! 🍡